test/new-e2e/tests/otel/otlp-ingest/compose/otlp_ingest_compose.yaml

```diff
@@ -0,0 +1,26 @@
+version: "3.9"
+services:
+  agent:
+    container_name: datadog-agent
+    ports:
+      - "4317:4317"
+      - "4318:4318"
```

Comment:
I was a bit surprised that this wasn't impacted by https://github.com/DataDog/datadog-agent/issues/32947, maybe docker compose is different - which is a good thing